src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerCredentials.java

```diff
@@ -70,6 +72,12 @@ public String getClientKey() {
         return clientKey == null ? null : clientKey.getPlainText();
     }
 
+    @CheckForNull
+    @Restricted(NoExternalUse.class)
+    public Secret getClientKeySecret() {
```

Comment:
IMO better to just change the type of the original getter and constructor parameter. AFAIK they are not referred to from other plugins. (From a _test_ in `docker-workflow` but we do not care about that.)